draft-ietf-rtcweb-jsep.xml

```diff
@@ -2595,7 +2605,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           identical to that indicated in the initial offers section
           above, except that the "a=ice-options" line, with the
           "trickle" option as specified in
-          <xref target="I-D.ietf-ice-trickle"></xref>, Section 4, is
+          <xref target="I-D.ietf-ice-trickle"></xref>, Section 4,
+          and the "ice2" option as specified in
+          <xref target="RFC8445"></xref>, Section 10, is
```

Comment:
is->are

Reply:
I think the subject is 'the "a=ice-options" line', which is singular.